code-challenges.test.js

```diff
@@ -23,24 +28,115 @@ const people = [
 
 // b) Create the function that makes the test pass.
 
+// Pseudo
+// input: an array of objects
+// output: array with a sentence
+// - First, lets .map() over the array of objets to interact with the values and their keys. set it in a varible (arr) that we well later use as a return.
+// - while inside .map() lets play with this values.
+  // - First lets access the values in name key with value.name and then separate each value with .split(" ") **this syntax in .split() will split each value by the group of strings in the array.** Set it in a varible for ease access (nameKey)
+  // - then lets capitalize each value in the name key (nameKey), user .map() to iteraterate over these split values and have the first index be uppercase by calling on the first charecters index with .chartAt() and then .toUpperCase(). **charAt() will only return the value given** to make it stick lets bring back the characters we are not upper casing with .substring() we give it the index of the next character as a parameter and with .join(" ") **this syntax in .join() will join each value by group of strings into a single array.** Set in a varible for ease access (cap)
+  // - return the string interpolation of your key name values alreay capitalized (cap) the word "is" and the values of your ocupation key (values.occupation).  **use backticks for string interpolations** 
+// - Last, lets return back the iterated array of objest (arr)
+
+const itIs42 = (array) =>{
```

Comment:
Awesome job on this one! I love the breakdown in the pseudocode and the function! 💯